images/macos/provision/core/xcode.ps1

```diff
@@ -14,20 +14,28 @@ $os = Get-OSVersion
 $xcodeVersions = Get-ToolsetValue "xcode.versions"
 $defaultXcode = Get-ToolsetValue "xcode.default"
 [Array]::Reverse($xcodeVersions)
+$threadCount = 8
```

Comment:
There is a function to return the number of processors, may be it is better to use it?
![image](https://user-images.githubusercontent.com/16715858/104029255-ce40b680-51da-11eb-8456-901a5c97c98e.png)

Reply:
nitpick: `[Environment]::ProcessorCount` as a variant

Reply:
Yep, `[Environment]::ProcessorCount` is even better from readability side